templates/content/oe-theme-helper-in-page-navigation--with-image.html.twig

```diff
@@ -0,0 +1,31 @@
+{#
```

Comment:
I've moved related helper method from oe_theme_helper.module to oe_theme_content_organisation.module.
It is helper template similar field--bare.html.twig to avoid code duplication.